content/ja/handbook/engineering/architecture/design-documents/transfer_data/index.md

```
@@ -0,0 +1,152 @@
+| 種類            | 説明                                                        |
+| --------------- | ----------------------------------------------------------- |
+| `Repository`    | Git の `fetch` 操作（pull、clone）に関連するエグレスデータ  |
+| `Artifacts`     | ダイレクトおよびプロキシ経由のエグレスによるアーティファクト転送 |
+| `Pages`         | Pages のエグレス（Artifacts API に依存）                    |
+| `Packages`      | パッケージレジストリのエグレス                              |
+| `Registry`      | コンテナレジストリのエグレス                                |
+| `Uploads`       | オブジェクトストアのエグレス                                |
+
+各種類は異なる実装を持ち、個別に計測できますが、メタデータ／データ転送テレメトリの収集とその消費・可視化は、同一の抽象化の上に構築する必要があります。
+
+## 概要
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)